### PR TITLE
Adding TPN for azure storage related packages

### PR DESCRIPTION
**Pull request by bishal-pdMSFT**

### CODE REVIEW

File: Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt [hunk 1]
```diff
@@ -2,48 +2,138 @@
...
+106. validator (https://github.com/chriso/validator.js)
+107. verror (https://github.com/joyent/node-verror)
+108. VSTS-task-lib (https://github.com/Microsoft/vsts-task-lib)
+109. window-size (https://github.com/jonschlinkert/window-size)
+110. wordwrap (https://github.com/substack/node-wordwrap)
+111. xml2js (https://github.com/Leonidas-from-XIV/node-xml2js)
+112. xmlbuilder (https://github.com/oozcitak/xmlbuilder-js)
+113. xtend (https://github.com/Raynos/xtend)
+114. yallist (https://github.com/isaacs/yallist)
+115.yargs (https://github.com/yargs/yargs)
```
Comment (ds-ms) [inline Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt]: spacing seems to be inconsistent

File: Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt [hunk 2]
```diff
@@ -2,48 +2,138 @@
...
+81.	request (https://github.com/request/request)
+82.	repeat-string (https://github.com/jonschlinkert/repeat-string)
+83.	right-align (https://github.com/jonschlinkert/right-align)
+84. safe-buffer (https://github.com/feross/safe-buffer)
+85.	sax (https://github.com/isaacs/sax-js)
+86.	semver (https://github.com/npm/node-semver)
+87.	shelljs (https://github.com/arturadib/shelljs)
+		Includes:wrench-js
+88. sntp (https://github.com/hueniverse/sntp)
+89.	source-map (https://github.com/mozilla/source-map)
```
Comment (kasubram) [inline Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt]: source-map (https://github.com/mozilla/source-map) is mentioned twice.

File: Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt [hunk 3]
```diff
@@ -2,48 +2,138 @@
...
+91.	string_decoder (https://github.com/rvagg/string_decoder)
+92.	stringstream (https://github.com/mhart/StringStream)
+93.	strip-ansi (https://github.com/chalk/strip-ansi)
+94.	supports-color (https://github.com/chalk/supports-color)
+95.	tough-cookie (https://github.com/SalesforceEng/tough-cookie)
+96.	tunnel (https://github.com/koichik/node-tunnel)
+97.	tunnel-agent (https://github.com/mikeal/tunnel-agent)
+98.	tweetnacl (https://github.com/dchest/tweetnacl-js)
+99. source-map (https://github.com/mozilla/source-map)
+100. tunnel (https://github.com/koichik/node-tunnel)
```
Comment (kasubram) [inline Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt]: tunnel (https://github.com/koichik/node-tunnel) is mentioned twice

File: Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt [hunk 4]
```diff
@@ -2,48 +2,138 @@
...
+101. uglify-js (https://www.npmjs.com/package/uglify-js)
+		Includes:Acorn
+		Includes:parse-js
+102. uglify-to-browserify (https://github.com/ForbesLindesay/uglify-to-browserify)
+103. underscore (https://github.com/jashkenas/underscore)
+		Includes:Normalize.css
+104. util-deprecate (https://github.com/TooTallNate/util-deprecate)
+105. uuid (https://github.com/defunctzombie/node-uuid)
```
Comment (kasubram) [inline Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt]: It was uuid  (https://github.com/kelektiv/node-uuid)
the repo url is changing.. can you check why?

File: Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt (hunk 2 above)
Comment (bishal-pdMSFT) [reply Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt]: # resolved

File: Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt (hunk 3 above)
Comment (bishal-pdMSFT) [reply Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt]: # resolved

File: Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt (hunk 4 above)
Comment (bishal-pdMSFT) [reply Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt]: both are required. I will add another back,

File: Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt (hunk 1 above)
Comment (bishal-pdMSFT) [reply Tasks/JenkinsDownloadArtifacts/ThirdPartyNotices.txt]: # fixed

**Pull request merged**